config/settings/production.py

```diff
@@ -89,9 +90,11 @@
     },
 }
 SENTRY_CELERY_LOGLEVEL = env.int("DJANGO_SENTRY_LOG_LEVEL", logging.INFO)
+SHOULD_IGNORE_EXCEPTIONS = " ".join(sys.argv).endswith("run python manage.py shell")
 RAVEN_CONFIG = {
     "CELERY_LOGLEVEL": env.int("DJANGO_SENTRY_LOG_LEVEL", logging.INFO),
     "DSN": SENTRY_DSN,
+    "ignore_exceptions": ["*"] if SHOULD_IGNORE_EXCEPTIONS else [],
```

Comment:
Heißt das nicht "Ignorier alles wenn wir spezifische zu ignorierende exceptions angegeben haben."?

Wir wollen doch nur den einen Befehl extra ignorieren, oder verstehe ich das falsch?

Reply:
Ja tut es. Wenn man in der REPL hängt, lösen Exceptions keine Sentry Fehler aus. Das ist genau was wir brauchen. Die Fehler sind ja trotzdem im REPL zu sehen, aber so ein Typo muss eben nicht in Sentry gespeichert werden.

Reply:
Ja, tut mir Leid; war hier etwas vorschnell. Nach erneutem nachdenken macht's sind. Good to go.